<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountConnection</class>
 <widget class="QDialog" name="AccountConnection">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>302</width>
    <height>192</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>302</width>
    <height>192</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>302</width>
    <height>192</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="windowIcon">
   <iconset resource="app.qrc">
    <normaloff>:/images/live-jamming.png</normaloff>:/images/live-jamming.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#AccountConnection {
	background-image: url(:/images/background_connection.png);
	width:200%;
}
#AccountConnection QLineEdit{
	background: url(:/images/send_box_connection.png) transparent no-repeat;
	height:22px;
	color:white;
	border:none;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetDefaultConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <property name="verticalSpacing">
    <number>8</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="loginLabel">
     <property name="styleSheet">
      <string notr="true">#loginLabel {
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="loginLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>205</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="styleSheet">
      <string notr="true">#passwordLabel {
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="passwordLineEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="serverIpLabel">
     <property name="styleSheet">
      <string notr="true">#serverIpLabel {
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Server IP</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="serverIpLineEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="serverPortLabel">
     <property name="styleSheet">
      <string notr="true">#serverPortLabel {
	color: rgb(255, 255, 255);
}</string>
     </property>
     <property name="text">
      <string>Server Port</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="serverPortLineEdit"/>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <kerning>true</kerning>
      </font>
     </property>
     <property name="accessibleName">
      <string/>
     </property>
     <property name="accessibleDescription">
      <string/>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">#AccountConnection QPushButton {
background:url(:/images/btn.png) transparent no-repeat;
border:0px;
width:80px;
height:20px;
}
#AccountConnection QPushButton:hover {
background:url(:/images/btn_hover.png) transparent no-repeat;
border:0;
width:70px;
height:25px;
}
#AccountConnection QPushButton:pressed {
background:url(:/images/btn_hover.png) transparent no-repeat;
border:0;
width:70px;
height:25px;
}
</string>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="app.qrc"/>
 </resources>
 <connections/>
</ui>
